src/BaseRewardStreams.sol

```diff
@@ -389,6 +386,20 @@ abstract contract BaseRewardStreams is IRewardStreams, EVCUtil, ReentrancyGuard
         return distributions[rewarded][reward].totalClaimed;
     }
 
+    /// @notice Returns the reward token amount for a specific rewarded token and epoch.
+    /// @param rewarded The address of the rewarded token.
+    /// @param reward The address of the reward token.
+    /// @param epoch The epoch to get the reward token amount for.
+    /// @return The reward token amount for the rewarded token and epoch.
+    function rewardAmount(
```

Comment:
It's to follow the Solidity style guide.
https://docs.soliditylang.org/en/latest/style-guide.html#order-of-functions

Reply:
got it. I am not sure if following standards in such cases is recommended. if you have two overloaded functions, it would be better if they lived close to each other. otherwise it can cause more harm than good

Reply:
I agree it's subjective. Your call.